wuvt/models.py

```diff
@@ -39,7 +39,7 @@ class Page(db.Model):
     published = db.Column(db.Boolean, default=True, nullable=False)
 
 
-    def __init__(self, name, slug, content, published=True, menu=None):
+    def __init__(self, name, slug, content, published, menu=None):
```

Comment:
I feel it makes sense to have to check the box to actually publish something, rather than having it published by default. Safety first.